src/dsp_tools/utils/create_logger.py

```diff
@@ -27,8 +27,6 @@ def _make_handler(
         handler instance
     """
     formatter = logging.Formatter(fmt="{asctime} {filename: <30} l. {lineno: >4} {levelname: <8} {message}", style="{")
-    formatter.default_time_format = "%Y-%m-%d %H:%M:%S"
-    formatter.default_msec_format = "%s.%03d"
```

Comment:
We need these 2 lines, because they specify the format of the timestamp in the log file. Without this, the default formatting would be applied, but the default differs from the format the DSP-API uses. 

Back then, when I added these 2 lines, my goal was that the logs of DSP-API and DSP-TOOLS use the same time stamp format, so that if you investigate an incident, you can easily find the corresponding lines in both log files.